src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -64,6 +64,8 @@
 /**
  * Contains a Database Public API implementation.
  *
+ * Methods may throw a {@link NoDocumentException} if the database does not exist.
```

Comment:
Just checking, but is this not in a `@throws` clause because it's an unchecked exception?